### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jlaura**
## Description
This PR updates the README to include instructions for downloading the ISIS test data area that is now hosted on S3 using the rclone tool.

## Related Issue

Multiple where posters are lamenting not being able to build with tests.

## Motivation and Context

Since the rsync server is no longer working to serve the test data and we are transitioning to S3, this update is a natural requirement to inform users how to access the test data.

## How Has This Been Tested?

The instructions were beta tested by ASC and non-ASC contributors to this repository.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -619,6 +620,33 @@ For versions of ISIS prior to ISIS 4.1.0, please cd into `$ISIS3DATA` instead of
...
+
+1. Set S3 as both your storage type and storage provider
+2. Set us-west-2 as both your region to connect to and as the location constraint. 
+3. Everything else, just leave as the default. 
+
+Example output: https://gist.github.com/Kelvinrr/706bbd54b1c2c30d0ce3d12f7dcaa10a
+
+Once rclone is configured, simply run: `rclone sync remote:asc-isisdata/isis_testData/ $ISISTESTDATA`
+where:
+  - $ISISTESTDATA is the directory you want to download the test data to. Whatever directory you use, you want to set the environment variable $ISISTESTDATA to this directory
```
Comment (jessemapel) [inline README.md]: Add something under the ISIS Test Data header indicating that the makefile tests find the test data using the `ISISTESTDATA` environment variable. Then, you can remove this bullet.

**Comment by jessemapel** [comment]
Fixes #4397

File: README.md [hunk 2]
```diff
@@ -619,6 +620,33 @@ For versions of ISIS prior to ISIS 4.1.0, please cd into `$ISIS3DATA` instead of
     rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/voyager1 .
     rsync -azv --delete --partial isisdist.astrogeology.usgs.gov::isisdata/data/voyager2 .
 
+### ISIS Test Data 
+ISIS is comprised of two types of tests, custom Makefile based tests, and GTest based tests. Those that are GTest based, make economical use of data that exists on the ISIS3 repo along with the source, so no special data is required to run those other than the ISIS data area. The Makefile tests depends on a separate source of data that consists of a few gigabytes of input and expected output data used for testing ISIS applications. The total size of this test data decreases as we work towards converting Makefile tests to GTests.
```
Comment (rbeyer) [inline README.md]: "tests depend on"

**Comment by KrisBecker** [comment]
Would now be a good time to also update the [Requests for Comment](https://github.com/USGS-Astrogeology/ISIS3#requests-for-comment) section and add RFC8 (#4653)?

**Comment by jlaura** [comment]
This is ready for a re-review.

@KrisBecker - sure, but in a different PR. This update is just for test data information.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**